src/test/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceSourceConfigBuilder.java

```diff
@@ -123,9 +124,14 @@ public SalesforceSourceConfigBuilder setParent(String parent) {
     return this;
   }
 
+  public SalesforceSourceConfigBuilder setOperation(String operation) {
```

Comment:
Where is this method called?

Reply:
I haven't built any test cases for it. It was erroring out when I tried to build it so I added this get past the build.